app/views/user/_modal_privileges.html.haml

```diff
@@ -8,11 +8,7 @@
         %h4#modal-privileges-label.modal-title
           = raw t('views.actions.privilege', user: @user.screen_name)
       %ul.list-group.groups--list
-        = render 'user/modal_privileges_item', privilege: 'blogger', description: t('views.modal.privilege.blogger'), user: @user
```

Comment:
Please remove the code that assigns these

Reply:
Refresh :)